check.py

```diff
@@ -175,6 +175,31 @@ def do_asm2wasm_test():
               fail_with_error('wasm interpreter error: ' + err) # failed to pretty-print
             fail_with_error('wasm interpreter error')
 
+        # verify debug info
+        if 'debugInfo' in asm:
+          jsmap = 'a.wasm.map'
+          cmd += ['--binarymap-file', jsmap,
```

Comment:
the file is still standalone file, I just changed its output format -- binary wasm just contains the reference